src/components/Accordion/Accordion.tsx

```diff
@@ -49,23 +49,29 @@ const getElementsFromContentData = (data = []) => {
     const panel =
       typeof content.panel === "string" ? (
         <AccordionPanel
+          id={`${content.label}-panel`}
           key={index}
           dangerouslySetInnerHTML={{ __html: content.panel }}
         />
       ) : (
-        <AccordionPanel key={index}>{content.panel}</AccordionPanel>
+        <AccordionPanel id={`${content.label}-panel`} key={index}>
```

Comment:
`content.label` is not the best to use because a label can be more than one word. In that situation, it would render something like `id="Gerry Kellman-panel"` and white space is not allowed. I'm actually not sure what would happen but I think "Gerry" would take precedence and it'll ignore the rest of the string.

Reply:
Didn't realize about the white space. Thanks! However, I didn't use the id because the Accordion components are rendered in a map, so each one needs to be different. I updated to this: `${id}-panel-${index}`